### ISSUE TITLE
[Others] About assignment due date

### ISSUE THREAD

**Issue by jae-way** [goldshakil/ElectricalProgrammingCourse2020#69]
I have a question about the due date of our assignments.

While talking with my friends, I found that all Monday class's assignment is due 4/5.
However, My friend is on Tuesday class, and his assignment-2 is due 4/15.

Is it just the difference between classes or am I getting wrong information due to system error or something?  

Thanks for reading

**Comment by goldshakil** [comment]
All assignments upto now have no deadline penalty so it doesn't matter.
I will upload a video this weekend and I will explain all the issue you guys are having.

**Comment by jae-way** [comment]
Thanks for helping. 
To be precise, 1st~4th week's assignments do not have deadline(originally 4/5) right??

**Comment by goldshakil** [comment]
Yes no deadline for any of those assignments ..i will upload a video on this weekend for more details.

**Comment by goldshakil** [comment]
Yes no deadline for any of those assignments ..i will upload a video on this weekend for more details.
